scripts/web-build

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/bash
```

Comment:
should be /usr/bin/env bash or /bin/bash

Reply:
Thanks!